pkg/server/server.go

```diff
@@ -341,11 +386,11 @@ func (m *Server) Download(ctx context.Context, req *pb.FileDownloadRequest) (*pb
 	if len(paths) == 0 {
 		return nil, errors.New("empty paths")
 	}
-	params, ok := m.getHostParams(req.GetHost())
-	if !ok {
-		return nil, status.Errorf(codes.Internal, "params are not set")
+	params, err := m.getHostParams(req.GetHost(), nil)
```

Comment:
Maybe we should extend FileDownloadRequest/FileUploadRequest with HostParams? Or there is no such case?